src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 800;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: #3d4e61;
+  line-height: normal;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+form {
+  width: 100%;
+  max-width: calc(100% - 20px);
+  margin: 0 auto;
+}
+
+input[type="search"] {
+  width: 100%;
+  border: 1px solid #000;
+  border-radius: 5px;
+}
+
+[data-qa="big"] input[type="search"] {
+  width: 100%;
+  display: flex;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  overflow: hidden;
+}
+
+[data-qa="small"] input[type="search"] {
+  width: 100%;
+  height: 42px;
+  line-height: normal;
+  border-radius: 4px;
+  border: 1px solid hsl(210, 25%, 91%);
+  background: hsl(0, 0%, 100%);
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
```

Comment:
please pay attention on this point from checklist
![Screenshot from 2023-07-29 17-51-07](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/82436db5-6058-4ae7-a362-7e8b15e2ac2b)